### ISSUE TITLE
Barcode scanning not working

### ISSUE THREAD

**Issue by seAfnan** [mebjas/html5-qrcode#487]
I have tried. QR code scanning is OK but Barcode is not working in my project. Here is my js code...
```
<script>
  var html5QrcodeScanner = new Html5QrcodeScanner(
    "reader", { fps: 10, qrbox: 250, experimentalFeatures: { useBarCodeDetectorIfSupported: true } }
  );
      
  function onScanSuccess(decodedText, decodedResult) {
    console.log('Scan Result: '+ decodedText);
    console.log('Scan Result: '+ decodedResult);
    html5QrcodeScanner.clear();
  }

  function onScanError(errorMessage) {
    console.log('Scan Error: '+ errorMessage);
  }

  html5QrcodeScanner.render(onScanSuccess);
</script>
```

**Comment by ROBERT-MCDOWELL** [comment]
again, you must tell us which barcode, and upload an image, and provide all your OS and browser info

**Comment by seAfnan** [comment]
Any type of barcode, currently i am scanning UPC barcode on Windows 10 with front cam of laptop on Edge browser.

![image](https://user-images.githubusercontent.com/20145270/174482234-350bb6b4-6d23-455a-85a0-f2b7dab459d8.png)

**Comment by seAfnan** [comment]
> again, you must tell us which barcode, and upload an image, and provide all your OS and browser info

Any type of barcode, currently i am scanning UPC barcode on Windows 10 with front cam of laptop on Edge browser.

![image](https://user-images.githubusercontent.com/20145270/174482234-350bb6b4-6d23-455a-85a0-f2b7dab459d8.png)

**Comment by seAfnan** [comment]
> again, you must tell us which barcode, and upload an image, and provide all your OS and browser info

Any type of barcode, currently i am scanning UPC barcode on Windows 10 with front cam of laptop on Edge browser.

![image](https://user-images.githubusercontent.com/20145270/174482234-350bb6b4-6d23-455a-85a0-f2b7dab459d8.png)

**Comment by mebjas** [comment]
Will be fixed in `version 2.3.1`. Please track in this parent bug https://github.com/mebjas/html5-qrcode/issues/605

Closing this one.
